optprompt/optprompt.py

```diff
@@ -9,13 +11,10 @@ class OptionPrompterError(Exception):
 class OptionPrompter():
 
     def __init__(self, *args, config_files=None, section_header='defaults', **kwargs):
-        self.opts = list()
-        self.config_files = list()
-        self.config_opts = list()
+        self.opts = []
```

Comment:
Is [] better than list() for some reason?

Reply:
It's just more pythonic. -- https://stackoverflow.com/questions/5790860/and-vs-list-and-dict-which-is-better